src/System.Text.Json/System/Text/Json/JsonParseObject.cs

```diff
@@ -182,6 +182,19 @@ public bool TryGetValue(string propertyName, out JsonObject value)
             }
         }
 
+        public int Length
+        {
+            get
+            {
+                var record = Record;
+                if (record.Type != JsonValueType.Array)
+                {
+                    throw new NullReferenceException();
```

Comment:
I copied this from TryGetValue just to be consistent. Should I change it also there?